src/ort_genai_c.cpp

```diff
@@ -108,7 +108,7 @@ OgaResult* OGA_API_CALL OgaGeneratorParamsSetSearchBool(OgaGeneratorParams* gene
 OgaResult* OGA_API_CALL OgaGeneratorParamsTryGraphCaptureWithMaxBatchSize(OgaGeneratorParams* generator_params, int32_t max_batch_size) {
   OGA_TRY
   auto* params = reinterpret_cast<Generators::GeneratorParams*>(generator_params);
-  params->max_batch_size = max_batch_size;
+  params->TryGraphCapture(max_batch_size);
```

Comment:
>TryGraphCapture [](http://example.com/codeflow?start=10&length=15)

OgaGeneratorParamsTryGraphCaptureWithMaxBatchSize can be called for CPU EP. in this case, it should does nothing